test/Docker.DotNet.Tests/QueryStringTests.cs

```diff
@@ -9,18 +9,18 @@ public class QueryStringTests
         [Fact]

```

Comment:
Can we add a test case for

`var p = new ServicesListParameters { Filters = new ServiceFilter() };`

Want to make sure we handle the null enumerable correctly.

Reply:
New test added